src/components/FlowRunLogs.vue

```diff
@@ -61,7 +62,10 @@
   }))
 
   const api = useWorkspaceApi()
-  const logsSubscription = usePaginatedSubscription(api.logs.getLogs, [logsFilter], { interval: 5000 })
+  const options = {
+    interval: isRunningStateType(props.flowRun.stateType) ? 5000 : Infinity,
```

Comment:
I'd expect `undefined` rather than `Infinity` to be better in that it would never kick off a timer that goes indefinitely. Didn't look at the `interval` implementation though.

Reply:
`Infinity` is the default value for `interval`. But I think `undefined` is probably easier to understand here. Will update 👍